packages/webapp/src/inductions/hooks/index.ts

```diff
@@ -0,0 +1,18 @@
+import { useMemo } from "react";
+import { useUALAccount } from "_app";
+import { getInductionUserRole } from "inductions";
+import { Endorsement, Induction, InductionRole } from "../interfaces";
+
+export const useInductionUserRole = (
+    endorsements: Endorsement[],
+    induction?: Induction
+): InductionRole => {
+    const [ualAccount] = useUALAccount();
+    const userRole: InductionRole = useMemo(
+        () => getInductionUserRole(endorsements, ualAccount, induction),
+        [ualAccount, induction, endorsements]
+    );
+    return userRole;
+};
+
+export * from "./queries";
```

Comment:
I like this query + hook refactor. But I would not mix logic + exports on the index file. All of our indexes files are exports. If you think they are small splitting them away would be silly we should remove the hooks folder and put them together in a hooks.ts file, until it grows and we can then create a folder (pretty much like you did with the `induction-journeys` which I love it :) )